packages/map-toolkit/src/index.css

```diff
@@ -0,0 +1,19 @@
+/*
```

Comment:
Since we don't intend to export this as part of this package (right?) can we implement this just where we need it (I think just in Storybook)